src/Tasks/Microsoft.Build.Tasks.csproj

```diff
@@ -984,6 +982,7 @@
     <PackageReference Include="System.Reflection.Metadata" />
     <PackageReference Include="System.Reflection.TypeExtensions" />
     <PackageReference Include="System.Resources.Writer" />
+    <PackageReference Include="System.Security.Principal.Windows" />
```

Comment:
No longer needed, but didn't get cleaned up when removing the other package changes.